### ISSUE TITLE
Could not find io.ktor:ktor-client-websocket:1.2.2.

### ISSUE THREAD

**Issue by RifqiFadhell** [Yazan98/Reflex#1]
Execution failed for task ':compileKotlin'.
> Could not resolve all files for configuration ':compileClasspath'.
   > Could not find io.ktor:ktor-client-websocket:1.2.2.
     Searched in the following locations:
       - file:/Users/rifqifadhl/.m2/repository/io/ktor/ktor-client-websocket/1.2.2/ktor-client-websocket-1.2.2.pom
       - file:/Users/rifqifadhl/.m2/repository/io/ktor/ktor-client-websocket/1.2.2/ktor-client-websocket-1.2.2.jar

do you have any solving ?

**Comment by Yazan98** [comment]
> Execution failed for task ':compileKotlin'.
> 
> > Could not resolve all files for configuration ':compileClasspath'.
> > Could not find io.ktor:ktor-client-websocket:1.2.2.
> > Searched in the following locations:
> > - file:/Users/rifqifadhl/.m2/repository/io/ktor/ktor-client-websocket/1.2.2/ktor-client-websocket-1.2.2.pom
> > - file:/Users/rifqifadhl/.m2/repository/io/ktor/ktor-client-websocket/1.2.2/ktor-client-websocket-1.2.2.jar
> 
> do you have any solving ?

Yes sorry for late .. you can replace the dependency with this version
implementation group: 'io.ktor', name: 'ktor-client-websocket', version: '1.1.4'
